app/frontend/src/cljs/teet/common/common_controller.cljs

```diff
@@ -122,7 +124,50 @@
           {:tuck.effect/type :navigate
            :page page
            :params params
-           :query (assoc query param value)})))
+           :query (assoc query param value)}))
+
+  ResponseError
+  (process-event [{err :err} app]
+    (case (-> err ex-data :error)
+      :authorization-failure
+      (do
+        (reset! api-token nil)
+        (t/fx (-> app
+                  (dissoc :user)
+                  (snackbar-controller/open-snack-bar (tr [:error :authorization-failure])
+                                                      :warning))
+              {:tuck.effect/type :navigate
+               :page :login}))
+
+      :server-error
+      (snackbar-controller/open-snack-bar app (tr [:error :server-error]) :error)
```

Comment:
Maybe later we should add custom error messages `[:error :server-error :my-custom-error]`.